Dockerfile.tester

```
@@ -48,6 +42,8 @@ RUN . /opt/ros/jazzy/setup.sh \
 FROM base
 
 ARG DATASET_NAME
```

Can this be set to default as "ipd"?

```suggestion
ARG DATASET_NAME=ipd
```

---

Embedding these into the image is causing a challenge as it gets baked into the command as well. In working on the automation I have been manually overriding the DATASET_NAME and building the image locally to get it working quickly instead of trying to override the full CMD. I think that the CMD is getting baked with the build ENV, not the runtime ENV.